resources/logical_interconnect_group.rb

```diff
@@ -38,15 +40,57 @@ def load_lig(item)
         raise "#{up[:networkType]} #{network_name} not found" unless net.retrieve!
         up.add_network(net)
       end
-      uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
+      parsed_uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
       item.add_uplink_set(up)
     end
     item
   end
+
+  # like? function specific to the interconnectMapTemplate resource
+  def interconnect_map_like?(item, another_item)
```

Comment:
It is not the case, there is some attributes that OneView adds to the LIG after the creation, so just creating method that deals with hash or arrays won't do the trick, it needs a specific filter to classify what are the configurations that need to be compared.

I don't know if we could add anything else in the general `like?` method right now. For example, to deal with arrays inside the LIG I don't care about how it is ordered (so I `sort`them), but inside another resource the order may be important and sorting may break the comparison, so these kind of changes may work for some resources but not for others since we don't have things like this mapped.